examples/TF_CONFIG/README.md

```diff
@@ -0,0 +1,91 @@
+conveniently orchestrate distributed trainings configured using
+the TF_CONFIG environment variable. This
+[MSG-GAN training application](https://github.com/roscisz/dnn_training_benchmarks/tree/master/TensorFlowV2_MSG-GAN_Fashion-MNIST)
+was used for the example.
+
+## Running the training without TensorHive
+
+In order to run the training manually, a separate process `python train.py`
+has to be run on each node with the appropriate values of parameters set as follows.
+
+**TF_CONFIG**
+
+The TF_CONFIG environment variable has to be appropriatly configured depending
+on the set of nodes taking part in the computations.
+For example, a training on two nodes gl01 and gl02 would require the following
+settings of TF_CONFIG:
+
+gl01:
+```bash
+TF_CONFIG='{"cluster":{"worker":["gl01:2222", "gl02:2222"]}, "task":{"type": "worker", "index": 0}}'
+```
+
+gl02:
+```bash
+TF_CONFIG='{"cluster":{"worker":["gl01:2222", "gl02:2222"]}, "task":{"type": "worker", "index": 1}}'
+```
+
+**Other environment variables**
+
+Depending on the environment, some other environment variables may have to be configured.
+For example, because our TensorFlow compilation uses a custom MPI library, the LD_LIBRARY_PATH environment
+variable has to be set for each process to /usr/mpi/gcc/openmpi-4.0.0rc5/lib/.
+
+**Choosing the appropriate Python version**
+
+In some cases, a specific Python binary has to be used for the training.
+For example, in our environment, a python binary from a virtual environment
+is used, so the python binary has to be defined as follows:
+
+```
+/home/roy/venv/p37avxmpitf2/bin/python
+```
+
+## Running the training with TensorHive
+
+The TensorHive `task nursery` module allows convenient orchestration of distributed trainings.
+It is available in the Tasks Overview view. The `CREATE TASKS FROM TEMPLATE` button allows to
+conveniently configure tasks supporting a specific framework or distribution method. In this
+example we choose the Tensorflow - TF_CONFIG template, and click `GO TO TASK CREATOR`:
+
+![choose_template](https://github.com/roscisz/TensorHive/tree/master/examples/TF_CONFIG/img/choose_template.png)
+
+In the task creator, we set the Command to
+```
+/home/roy/venv/p37avxmpitf2/bin/python /home/roy/dnn_training_benchmarks/TensorFlowV2_MSG-GAN_Fashion-MNIST/train.py
+```
+
+In order to add the LD_LIBRARY_PATH environment variable, we specify the parameter name,
+select Static (the same value for all processes) and click `ADD AS ENV VARIABLE TO ALL TASKS`:
+
+![env_var](https://github.com/roscisz/TensorHive/tree/master/examples/TF_CONFIG/img/env_var.png)
+
+Then, set the appropriate value of the environment variable (/usr/mpi/gcc/openmpi-4.0.0rc5/lib/).
+
+The task creator allows also to conveniently specify other command-line arguments. For example,
+to specify batch size, we specify parameter name --batch_size, again select Static and click
+`ADD AS PARAMETER TO ALL TASKS` and specify its value (in our case 32).
```

Comment:
A lot of `specify` word in this sentence. Maybe change to something like " we **enter** parameter name" and "and **set** its value"

Reply:
fixed in c739db6cf98760d96c120aaf735e5f0036b5c3e4